README.md

```diff
@@ -21,6 +21,8 @@ NCL to Python consists of pages that explain specifically how to achieve somethi
 in NCL in Python, including any algorithmic differences, guidance regarding replication under different
 conditions or circumstances, and any other relevant comparisons between the NCL and Python functionality
```

Comment:
```suggestion
conditions or circumstances, and any other relevant comparisons between the NCL and Python functionality.
```

Reply:
Actually this whole page is missing periods at the end of paragraphs. Is that a stylistic choice?

Reply:
It was, but I can add them 👍